.github/workflows/bootstrap.yaml

```diff
@@ -4,9 +4,6 @@ on:
   pull_request:
     branches:
       - '*'
-  push:
-    branches:
-      - '*'
```

Comment:
Not technically related, but these test runs did not even work, yet had absolutely no consequence.